subsys/net/lib/sockets/sockets.c

```diff
@@ -192,6 +192,28 @@ Z_SYSCALL_HANDLER(zsock_close, sock)
 }
 #endif /* CONFIG_USERSPACE */
 
+int _impl_zsock_shutdown(int sock, int how)
+{
+	/* shutdown() is described as just disabling recv() or send()
```

Comment:
change 'or' to 'and/or' since SHUT_RDWR disables both.

Should we support at least SHUT_RDWR? I guess you could put down the net_context below.

Reply:
> Should we support at least SHUT_RDWR? I guess you could put down the net_context below.

I doubt that. After shutdown(), soon close() is called, and we do it there. Eventually, we should implement shutdown() completely, but it largely has implications for TCP, so first we need proper TCP implementation.

Reply:
> change 'or' to 'and/or' since SHUT_RDWR disables both.

done.